frontend/src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,43 @@
+import { Wrap, UL, Li, MobileStepperLabel } from './styled';
+
+export const Step = (props) => {
```

Comment:
mě přijde možná trochu zbytečná tahle komponenta a tam kde jí máš použitou bych dal rovnou to `Li` -> takže místo toho `<Step>`